uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java

```diff
@@ -439,6 +440,18 @@ public void addHostName(final String protocol, String s) {
         fullHostNames.put(protocol, s);
     }
 
+    public void updateCacheWithHostNames() {
+        JGitFileSystemsCache fc = fsManager.getFsCache();
+        List<JGitFileSystem> jGitFileSystems = fc.getFileSystems().stream()
+                .map(fsName -> ((JGitFileSystemProxy) fsManager.get(fsName)).getRealJGitFileSystem()).collect(Collectors.toList());
```

Comment:
Just a suggestion.
You could replace the `collect() `by a `forEach` with the content of the for loop below.
This way you don't need to create a `List<JGitFileSystem>`.

Reply:
done,Nice catch. Thanks @caponetto